Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM ghcr.io/sscpac/statick:0.7.2
+FROM ghcr.io/sscpac/statick:0.8.1
 
 LABEL "name"="Statick Action"
-LABEL "version"="0.7.2"
+LABEL "version"="0.8.1"
```

Comment:
If this is going to support the `--timings` flag should the version go to `0.9.0`?

Reply:
df43ff6    Still needs testing, as 0.9.0 has not be published yet.